package.json

```diff
@@ -35,8 +36,7 @@
       "git add README.md"
     ],
     "*.{ts,tsx}": [
-      "prettier-standard",
-      "tslint --fix",
+      "prettier-eslint --write",
```

Comment:
Is this one actually needed? Isn't it easier to go simply with
```
eslint --fix
prettier --write
```
? :)

Reply:
@dannytce prettier is known for having conflicting issues when run after ESLint, which would ultimately mean the code would change twice and end-up on the prettier version.

`prettier-eslint` is basically a shortcut to run prettier, but respecting any conflicting rules from ESLint config.

Reply:
So either you go with `prettier-eslint` or extending your ESLint config with `eslint-config-prettier`, right? :)